app/assets/javascripts/admin/addon/components/themes-grid-placeholder.gjs

```diff
@@ -28,59 +38,140 @@ export default class ThemesGridPlaceholder extends Component {
     }
   }
 
-  <template>
-    <svg viewBox="0 0 636 347" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <rect
-        width="635.115"
-        height="347"
-        fill={{htmlSafe this.themeColors.secondary}}
-      />
-      <path
-        d="M54.9766 79.1039C55.9198 76.0065 59.8623 75.0916 62.0732 77.4571L121.448 140.986C123.659 143.351 122.48 147.223 119.326 147.955L34.621 167.611C31.467 168.343 28.7034 165.386 29.6466 162.288L54.9766 79.1039Z"
-        fill={{htmlSafe this.themeColors.primary}}
-      />
-      <path
-        d="M398.487 211.02C400.651 208.611 404.611 209.448 405.615 212.527L432.579 295.196C433.584 298.274 430.879 301.285 427.711 300.615L342.635 282.633C339.467 281.963 338.212 278.115 340.376 275.707L398.487 211.02Z"
-        fill={{htmlSafe this.themeColors.tertiary}}
-      />
-      <circle cx="109.357" cy="262.879" r="44.1636" fill="#D1F0FF" />
-      <circle cx="365.927" cy="103.048" r="44.1636" fill="#E45735" />
-      <rect
-        x="166.139"
-        y="68.751"
-        width="81.9226"
-        height="81.9226"
-        rx="4.20606"
-        transform="rotate(-15.9297 166.139 68.751)"
-        fill={{htmlSafe this.themeColors.danger}}
-      />
-      <rect
-        x="500.521"
-        y="100.296"
-        width="81.9226"
-        height="81.9226"
-        rx="4.20606"
-        transform="rotate(-15.9297 500.521 100.296)"
-        fill={{htmlSafe this.themeColors.success}}
-      />
-      <rect
-        x="481.857"
-        y="222.921"
-        width="121.976"
-        height="54.6788"
-        rx="4.20606"
-        transform="rotate(9.12857 481.857 222.921)"
-        fill={{htmlSafe this.themeColors.love}}
-      />
-      <rect
-        x="176.654"
-        y="240.608"
-        width="121.976"
-        height="54.6788"
-        rx="4.20606"
-        transform="rotate(-22.7296 176.654 240.608)"
-        fill={{htmlSafe this.themeColors.primary}}
-      />
-    </svg>
-  </template>
+  get safeThemeColors() {
+    let colors = this.themeColors;
+
+    return {
+      primary: htmlSafe(colors.primary),
+      secondary: htmlSafe(colors.secondary),
+      tertiary: htmlSafe(colors.tertiary),
+      quaternary: htmlSafe(colors.quaternary),
+      highlight: htmlSafe(colors.highlight),
+      danger: htmlSafe(colors.danger),
+      success: htmlSafe(colors.success),
+      love: htmlSafe(colors.love),
+    };
+  }
+
+  get gradientId() {
+    return `bgGradient-${this.args.theme.id}-${this.randomVariant}`;
+  }
 }
+
+setComponentTemplate(
+  hbs`
```

Comment:
@ellaestigoy any idea know how did this get added here? 😃

Reply:
@CvX Ha yeah, I was poking around for how to inline templates in Glimmer components, and this is what seemed to work. Happy to change it if there’s a better or more current approach 😄

Reply:
Using the `<template>` tag, as this file did before, is the how templates are embedded in gjs.